app/models/concerns/users/roles.rb

```diff
@@ -86,6 +86,10 @@ def roles_has_changed?
     roles_changed || organization_roles.any?(&:changed?)
   end
 
+  def has_root_user?
+    root unless root == self
+  end
```

Comment:
Esto si no lo vas a usar como pregunta, sería solo `root_user`, para no hacerla larga lo hago yo.
```suggestion
  def parent_root
    root unless root == self
  end
```